aiida/manage/external/postgres.py

```diff
@@ -23,8 +23,6 @@
 if TYPE_CHECKING:
     from aiida.manage.configuration import Profile
 
-__all__ = ('Postgres', 'PostgresConnectionMode', 'DEFAULT_DBINFO')
```

Comment:
Again, this is in principle breaking API change, but these are only used in a couple of places, chiefly in `verdi quicksetup` command.